io.github.phastmike.tags.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "io.github.phastmike.tags",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "tags",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "tags",
```

Comment:
GtkFileChooserDialog does not support portals https://github.com/phastmike/tags/blob/0b4ad97a177c08056bee075de65635304782361c/src/window.vala#L313, it is also deprecated

You should use [GtkFileChooserNative](https://docs.gtk.org/gtk4/class.FileChooserNative.html) (deprecated) or [GtkFileDialog](https://docs.gtk.org/gtk4/class.FileDialog.html)

![image](https://github.com/flathub/flathub/assets/62639087/22cdcfe7-2e7b-4c30-8ee0-65743e3f4193)


If you can't work on this immediately give some limited filesystem access.

Reply:
This one is a bit problematic, I've tried to use GtkFileDialog but found a problem on the portal as indicated here:

https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/issues/84

The issue has been open for quite some time.

This application only makes sense if you can use your system files as it is a system tool to analyse logs.

Not being able to analyse system/home files defeats the app's purpose.

I'll need to add permissions, at least for home folder :/

Reply:
Been checking the filesystem permissions and in my particular case, i need it to have host permissions to access some network shares that are mapped into /mnt. I think that we should go with the Risky label on the app and let user decide if he's willing to try the app, similar to gedit flatpak (e,g.).